script/arbitrum/utils/concentrators/curve/InitFraxBPGlp.sol

```diff
@@ -1,123 +1,137 @@
 // SPDX-License-Identifier: MIT
 pragma solidity 0.8.17;
 
-// import "src/arbitrum/concentrators/curve/CurveGlpConcentrator.sol";
-// import "script/arbitrum/utils/InitBase.sol";
-// import "src/arbitrum/utils/FortressArbiSwap.sol";
-// import "src/arbitrum/utils/FortressArbiRegistry.sol";
-contract InitFraxBPGlp {}
-// contract InitFraxBPGlp is InitBaseArbi {
+import "src/arbitrum/concentrators/curve/CurveGlpConcentrator.sol";
+import "script/arbitrum/utils/InitBase.sol";
+import "src/arbitrum/utils/FortressArbiSwap.sol";
+import "src/arbitrum/utils/FortressArbiRegistry.sol";
+import "src/arbitrum/utils/CurveArbiOperations.sol";
+
+contract InitFraxBPGlp is InitBaseArbi {
     
-//     function _initializeFraxBPGlp(address _owner, address _fortressArbiRegistry, address _fortressSwap, address _platform, address _compounder) public returns (address) {
+    function _initializeFraxBPGlp(address _owner, address _fortressArbiRegistry, address _fortressSwap, address _platform, address _compounder, address _ammOperations) public returns (address) {
        
-//         _initSwapFraxBPGlp(_fortressSwap);
+        _initSwapFraxBPGlp(_fortressSwap);
         
-//         // ------------------------- init TriCrypto compounder -------------------------
+        // ------------------------- init TriCrypto compounder -------------------------
 
-//         uint256 _convexPid = 5;
-//         uint256 _poolType = 1; 
+        uint256 _convexPid = 10;
```

Comment:
Can you explain what the convex pid here is and why it changed from 5 to 10?

Reply:
It's ID of Curve LP token in Convex stacking contract. It was updated by Convex a few weeks ago.

Reply:
Say we run our vaults and they change the ID again. How do we notice and what will happen until we fix it on our side?